packages/atomic/src/components/result-template-components/result-template-decorators.tsx

```diff
@@ -55,6 +55,14 @@ export function ResultContext(opts: {folded: boolean} = {folded: false}) {
       return connectedCallback && connectedCallback.call(this);
     };
 
+    component.componentWillRender = function () {
```

Comment:
Why is this now needed ?

Reply:
Found a bug (error in console) where something was executed in this method where as the component did not render since there was an error.

Just a slight improvement